main.js

```diff
@@ -0,0 +1,69 @@
+const bookList = document.querySelector('.book-list');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const form = document.querySelector('#form');
+let booksStore = [];
+
+// getting existing data from localStorage
+let existingData = JSON.parse(localStorage.getItem('data'));
+if (existingData == null) existingData = booksStore;
+
+// adding existingData to booksStore
+booksStore = booksStore.concat(existingData);
+
+// displaying data in the ui
+booksStore.forEach((item) => {
+  bookList.innerHTML += `
+<div class="book" id ="${item.title}">
+      <p>${item.title}</p>
+      <p>${item.author}</p>
+      <button type="button" class="remove">Remove</button>
+      <hr>
+    </div>
+`;
+});
+
+function handleSubmit(e) {
+  e.preventDefault();
+  const title = bookTitle.value;
+  const author = bookAuthor.value;
+  const bookDescription = {
+    title,
+    author,
+  };
+
+  // setting the inputValue to empty
+  bookTitle.value = '';
+  bookAuthor.value = '';
+
+  // add new book
+  if (title && author !== '') {
+    booksStore.push(bookDescription);
```

Comment:
## Optional

- [ ] I suggest you use array methods that don't mutate the array. For Example, `concat` is one of the best choices instead of `push`